### ISSUE TITLE
Allow line feeds and comments before file level directives

### ISSUE THREAD

**Issue by koalaman** [koalaman/shellcheck#779]
#### For bugs
- Rule Id (if any, e.g. SC1000): any
- My shellcheck version (`shellcheck --version` or "online"): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh

# SC2030: Modification of WINE is local (to subshell caused by (..) group).
# SC2031: WINE was modified in a subshell. That change might be lost
# shellcheck disable=SC2030,SC2031

true
true | WINE=foo
echo $WINE
```

#### Here's what shellcheck currently says:

```
$ shellcheck myscript
 
Line 8:
true | WINE=foo
       ^-- SC2030: Modification of WINE is local (to subshell caused by pipeline).
 
Line 9:
echo $WINE
     ^-- SC2031: WINE was modified in a subshell. That change might be lost.

```

#### Here's what I wanted or expected to see:

Nothing. This should count as a top level directive.

**Comment by koalaman** [comment]
Fixed in f260381
